dts/arm/nordic/nrf52832.dtsi

```diff
@@ -212,6 +214,7 @@
 			reg = <0x40024000 0x1000>;
 			interrupts = <36 1>;
 			status = "ok";
+			prescaler = <0>;
```

Comment:
Why are the prescaler entries missing from the TIMER nodes, below?

Reply:
fixed